integration-tests/src/test/resources/features/source-slack.feature

```diff
@@ -0,0 +1,62 @@
+Feature: Slack Source tests
+
+  Background:
+    Given authenticate against Manager
+    And create a new Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "ready" within 4 minutes
+    And the Ingress of Bridge "mybridge" is available within 2 minutes
+
+  @slacksource
+  Scenario: Slack Source Processor is created and slack message as source should result into other generated slack message
+    When add a Processor to the Bridge "mybridge" with body:
+    """
+    {
+      "name": "slackSourceProcessor",
+      "source": {
+        "type": "Slack",
+        "parameters": {
+            "channel": "${env.slack.channel.name}",
+            "token": "${env.slack.webhook.token}"
+          }
+      },
+      "filters": [
+        {
+          "key": "data.text",
+          "type": "StringContains",
+          "values": ["Slack Event Source Feature trigger"]
+        }
+      ]
```

Comment:
What is the purpose of this filter instead?

Reply:
We uses the same *mc* slack channel for multiple tests. If you check https://app.slack.com/client/T033KAWDGUC/C0346PP8EL8 there are also messages like "hello world ..."

So in my test I want to listen only for specific messages on *mc* slack channel